CMakeLists.txt

```diff
@@ -271,19 +271,31 @@ endif()
 # =============================================================================
 # Add project directories AFTER CMake modules
 # =============================================================================
+
+if(READLINE_FOUND)
+  # MAC libedit.3.dylib does not have rl_event_hook
+  # Readline_LIBRARY may be a binary lib or (on the MAC) a tbd file that
+  # points to the library and mentions all its definitions.
+  # this part has to be prior to adding the nrniv subdirectory.
+  execute_process(COMMAND grep -q rl_event_hook ${Readline_LIBRARY} RESULT_VARIABLE result)
+  if(NOT result EQUAL 0)
+    # define for src/oc/hoc.c
+    set(DEF_RL_GETC_FUNCTION use_rl_getc_function)
+  endif()
+  message(STATUS "READLINE_FOUND and DEF_RL_GETC_FUNCTION=${DEF_RL_GETC_FUNCTION}")
```

Comment:
```suggestion
```
I would remove this because `Readline found` is already printed by CMake and this extra information is more for developer / debugging.